sdk/go/auto/workspace.go

```diff
@@ -183,11 +183,18 @@ type StackSummary struct {
 	URL              string `json:"url,omitempty"`
 }
 
+// TokenInformation contains information about the currently logged-in Pulumi access token.
+type TokenInformation struct {
+	Name         string `json:"name"`
+	Organization string `json:"organization"`
+}
+
 // WhoAmIResult contains detailed information about the currently logged-in Pulumi identity.
 type WhoAmIResult struct {
-	User          string   `json:"user"`
-	Organizations []string `json:"organizations,omitempty"`
-	URL           string   `json:"url"`
+	User             string           `json:"user"`
+	Organizations    []string         `json:"organizations"`
+	URL              string           `json:"url"`
+	TokenInformation TokenInformation `json:"tokenInformation"`
```

Comment:
So I think we'll need `omitempty` here, since token information isn't always returned (e.g. if you are using a DIY backend). We could also therefore make this `*TokenInformation` so that we can have `nil` for the empty state. Moreover, it looks like you've removed `omitempty` from `Organizations` (which can also be not present) -- we'll want to put that back.

Reply:
Done.